front/pages/poke/templates/[tId].tsx

```diff
@@ -279,6 +279,11 @@ function TemplatesPage({
     );
   }
 
+  if (Object.keys(form.formState.errors).length > 0) {
+    // Useful to debug in case you have an error on a field that is not rendered
```

Comment:
I would say let's never log client-side in production ?

Reply:
no strong opinion here

Reply:
We're in Poké and I think it can be valuable for debugging purpose when you don't understand why your form won't submit without any error displayed. Merging as-is since you agreed IRL 🙇🏻‍♀️